modules/products/nodejs/src/main/kotlin/com/metalbear/mirrord/products/nodejs/NodeRunConfigurationExtension.kt

```diff
@@ -56,4 +72,22 @@ class NodeRunConfigurationExtension : AbstractNodeRunConfigurationExtension() {
     override fun isApplicableFor(profile: AbstractNodeTargetRunProfile): Boolean {
         return profile is NodeJsRunConfiguration
     }
+
+    override fun attachToProcess(
+        configuration: AbstractNodeTargetRunProfile,
+        handler: ProcessHandler,
+        runnerSettings: RunnerSettings?
+    ) {
+        val envsToRemove = runningProcessEnvs.remove(configuration.project) ?: return
```

Comment:
I think it'd be better to store the env before we apply mirrord env, then restore it instead of removing our keys - imagine the following scenario:
1. user adds an env "CAKE=true"
2. mirrord grabs "CAKE=false" from remote
3. post clean, there's no cake env :(

Reply:
great thought! ill fix (: